packages/blockchain-utils-validation/src/blockchains/ethereum.ts

```diff
@@ -64,6 +65,11 @@ async function validateErc1271Link(proof: LinkProof): Promise<LinkProof | null>
 }
 
 async function validateLink(proof: LinkProof): Promise<LinkProof | null> {
+  // Handle legacy CAIP links
+  if (proof.account) {
+    proof.account = normalizeAccountId(proof.account).toString()
```

Comment:
I'm a little nervous about this function modifying its input out from under the caller.  May be worth it to copy the `proof` object and then only modify the copy